persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/ReadSideActor.scala

```diff
@@ -117,6 +120,7 @@ private[lagom] class ReadSideActor[Event <: AggregateEvent[Event]](
     // Yes, we are active
 
     case Done =>
+      // This `Done` is materialization of the `Sink.ignore` above.
```

Comment:
While refactoring `TopicProducerActor` I've been doing some cleanup on `ReadSideActor` too.

Eventually, both actor could be refactored and common pieces extracted, but we'll fight this fight when we get there.